### ISSUE TITLE
Transfer schedule to new version

### ISSUE THREAD

**Issue by Frankyfit86** [YezerSTN/INSSIST#56]
i have installed the new version 26. thanks for your effort.
but now it is seen as a new standalone plugin.
and i still have schedule in version 25. but they are not taken over. i don't want to create the schedule again. is there any way to copy them into the new version? thanks

**Comment by YezerSTN** [comment]
I don't think it is possible, because it is an update as such, so the configuration is lost with each update. What you can do is to IMPORT A .CVS

For more information visit: https://gist.github.com/slashedio/34e1fe9b143183c9cde86070c85f532c
